index.html

```diff
@@ -44,129 +65,150 @@ <h4 class="date">Published on Feb 27, 2014</h4>
 
             <aside>
                 <img src="images/side-image-01.jpg">
-                <h3>Contributing to Open Source Part II: The Real Way</h3>
-                <small>The Odin Project</small>
-                <small>29,985 views</small>
+                <p><strong>Contributing to Open Source Part II: The Real Way</strong><br>
+                    <span class="text-font"><small>The Odin Project<br>
+                    29,985 views </small></span>
+                <p>
+                
             </aside>
             
             <aside>
                 <img src="images/side-image-02.jpg">
-                <h3>'How to Get a Job at the Big 4 - Amazon, Facebook, Google & Microsoft' by</h3>
-                <small>imtiana</small>
-                <small>929,985 views</small>
+                <p><strong>'How to Get a Job at the Big 4 - Amazon, Facebook, Google & Microsoft' by</strong><br>
+                    <span class="text-font"><small>imtiana<br>
+                    929,985 views </small></span>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-03.jpg">
-                <h3>Programming in Visual Basic .Net How to Connect Access Database to</h3>
-                <small>iBasskung</small>
-                <small>16,129,985 views</small>
+                <p><strong>Programming in Visual Basic .Net How to Connect Access Database to</strong><br>
+                    <span class="text-font"><small>iBasskung<br>
+                    16,129,985 views</small></span>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-04.jpg">
-                <h3>REST API concepts and examplesy</h3>
-                <small>WebConcepts</small>
-                <small>3,529,985 views</small>
+                <p><strong>REST API concepts and examplesy</strong><br>
+                    <span class="text-font"><small>WebConcepts<br>
+                    3,529,985 views</small></span>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-05.jpg">
-                <h3>GitHub - Why Microsoft Paid $7.5B for the Future of Software! - A Case</h3>
-                <small>Valuetainment</small>
-                <small>444,985 views</small>
+                <p><strong>GitHub - Why Microsoft Paid $7.5B for the Future of Software! - A Case</strong><br>
+                    <span class="text-font"><small>Valuetainment<br>
+                    444,985 views</small></span>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-06.jpg">
-                <h3>How to create a 3D Terrain with Google Maps and height maps in Photoshop</h3>
-                <small>Orange Box Ceo</small>
-                <small>29,296 views</small>
+                <p><strong>How to create a 3D Terrain with Google Maps and height maps in Photoshop</strong><br>
```

Comment:
Don't use the <strong> tag to make regular text bold, use CSS instead. [The HTML Strong Importance Element (<strong>) indicates that its contents have strong importance, seriousness, or urgency. Browsers typically render the contents in bold type.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong)